okhttp-testing-support/build.gradle

```diff
@@ -2,14 +2,9 @@ dependencies {
   api project(':okhttp')
   api deps.junit
   api deps.assertj
-  api deps.bouncycastledebug
-  implementation deps.bouncycastlepkix, {
-    // prefer debug build for our own testing
-    exclude module:"bcprov-jdk15on"
-  }
-  implementation deps.bouncycastletls, {
-    exclude module:"bcprov-jdk15on"
-  }
+  api deps.bouncycastle
```

Comment:
I haven't used the debug to date, so giving up on it.  More viable to test in a separate project.